requirements.txt

```diff
@@ -0,0 +1 @@
+GitPython
```

Comment:
Can we make this an optional requirement? We should be able to conditionalize support if the module exists or not